docs/standards.md

```diff
@@ -207,6 +208,36 @@ Things to take note of:
 
 `TODO`
 
+## Batch Scripts
+
+There is a set of batch scripts written in [batch_scripts.py](../scripts/batch_scripts.py). The motivation for these scripts is to provide crucial functionality for the hackathon process that is lacking in the admin UX. The goal is to extend the functionality to provide a full-fledged CLI client so that we can interface with the API without using the admin dashboard.
+
+### How to Run
+
+This script expects python3, and will not work in python2. Before you run the script, make sure that you have installed the following library:
+
+* [bson](https://pypi.org/project/bson/): `pip install bson`
+
+Then, run the following command:
+```python3 batch_scripts.py```
+
+In order for the script to work with all of the actions, you must have a staff account. Otherwise, it's possible that you do not have the proper permission to execute all of the actions.
+
+### Existing Actions in the Script file
+
+There are 6 batch actions:
+
+1. `updateStatus`: Update the status of a set of hackers. The script asks for a file which contains the IDs of hackers to update, and also for a pre-requisite status that the hacker must be in order for the update to properly go through.
```

Comment:
It would be nice to describe the format of the file like below in `invite users`. This also applies for `dayOf`, `weekOf`, and `downloadResume`. Are all the formats csv files, 1 entry per line, with a header?